r2-core/src/main/java/com/linkedin/r2/transport/http/client/ratelimiter/Rate.java

```diff
@@ -16,6 +16,9 @@
 
 package com.linkedin.r2.transport.http.client.ratelimiter;
 
+import java.util.Objects;
```

Comment:
Unused import?

Reply:
Between draft iterations, this one's missing the equals method that uses Objects. 😄